docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
```

Comment:
I've attempted to make them more uniform. Feel free to point out any other points where you dislike it.